frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/PortalConstants.java

```diff
@@ -649,6 +649,7 @@ public class PortalConstants {
     // Rest API constants
     public static final UserGroup API_WRITE_ACCESS_USERGROUP;
     public static final Boolean API_TOKEN_ENABLE_GENERATOR;
+    public static final Boolean DISABLE_REST_API_WRITE_ACCESS_TOKEN_IN_PREFERENCES;
```

Comment:
please rename this field to `REST_API_WRITE_ACCESS_TOKEN_IN_PREFERENCES`... in line with property key.